react-commenting.js

```diff
@@ -28,16 +28,16 @@ class App extends Component{
 
         <h3>Enter text here:</h3>
         <input
-          {/* 4. */}
+          {/* 4. create a class */}
           onChange={ this.robot }
```

Comment:
this one calls the `robot` method above as soon as changes are made to the input field